dts/bindings/serial/espressif,esp32-uart.yaml

```diff
@@ -21,3 +21,15 @@ properties:
       Overrides hw-flow-control if both are set.
       Using this mode, the pin assigned to DTR
       is asserted during transmission.
+
+  tx-invert:
+    type: boolean
+    description: |
+      Invert the binary logic of tx pin. When enabled, physical logic levels are inverted and
+      we use 1=Low, 0=High instead of 1=High, 0=Low.
+
+  rx-invert:
```

Comment:
these same properties were just added on an NXP controller in https://github.com/zephyrproject-rtos/zephyr/pull/71802/

should we make a generic property?

Reply:
Yes, I think we should make a generic property in uart driver in the future.

Reply:
why not updating the `uart-controller.yaml` now?

Reply:
Here is the PR https://github.com/zephyrproject-rtos/zephyr/pull/72947